src/Microsoft.ML.AutoML/Utils/UserInputValidationUtil.cs

```diff
@@ -248,6 +249,11 @@ private static void ValidateTrainDataColumn(IDataView trainData, string columnNa
             var nullableColumn = trainData.Schema.GetColumnOrNull(columnName);
             if (nullableColumn == null)
             {
+                var closestNamed = ClosestNamed(trainData, columnName, 2);
```

Comment:
Is `2` high enough?

I see `git` is using <7 for their limit ([code](https://github.com/git/git/blob/1c52ecf4ba0f4f7af72775695fee653f50737c71/help.c#L523-L524)) in correct command line arguments.

Reply:
Increased to 7.